h2/src/main/org/h2/command/dml/Query.java

```diff
@@ -397,7 +403,7 @@ public final ResultInterface query(int limit, ResultTarget target) {
         }
         fireBeforeSelectTriggers();
         if (noCache || !session.getDatabase().getOptimizeReuseResults() ||
-                (session.isLazyQueryExecution() && !neverLazy)) {
+                (session.isLazyQueryExecution() /* && !neverLazy*/)) {
```

Comment:
Why you commented `neverLazy` condition here?

Reply:
Typo, dirty change. Thanks.